recipes/rusty-cpp/all/conanfile.py

```diff
@@ -0,0 +1,53 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
+import os
+
+
+required_conan_version = ">=2.0"
+
+
+class RustyCppConan(ConanFile):
+    name = "rusty-cpp"
+    description = "Write C++ code like Rust!"
+    license = "Apache-2.0 OR MIT"
```

Comment:
```suggestion
    license = "Apache-2.0", "MIT"
```

let's wait until Conan 2.x has better modelling of licenses to make this OR decision on behalf of the consumer, safer to have the list approach for now unless upstream clearly states its SPDX identifier

Reply:
Tracked as https://github.com/conan-io/conan/issues/17446